ext/mini_racer_extension/extconf.rb

```diff
@@ -1,3 +1,8 @@
+if RUBY_ENGINE == "truffleruby"
+  File.write("Makefile", dummy_makefile($srcdir).join(""))
+  return
+end
+
 require 'mkmf'
```

Comment:
This needs to be the first line, `dummy_makefile` is defined by mkmf

Reply:
Fixed, `require "mkmf"` is the first line.